tests/src/test/scala/org/apache/openwhisk/core/controller/test/ControllerTestCommon.scala

```diff
@@ -73,7 +73,7 @@ protected trait ControllerTestCommon
   override lazy val entitlementProvider: EntitlementProvider =
     new LocalEntitlementProvider(whiskConfig, loadBalancer, instance)
 
-  override val activationIdFactory = new ActivationId.ActivationIdGenerator() {
+  override val activationIdFactory: ActivationId.ActivationIdGenerator = new ActivationId.ActivationIdGenerator {
```

Comment:
when the type is obvious (it's on the RHS) i think it's ok to omit.
```suggestion
  override val activationIdFactory = new ActivationId.ActivationIdGenerator {
```